### ISSUE TITLE
Option to check Style/YodaCondition only on equality operators

### ISSUE THREAD

**Issue by qnet-herwin** [bbatsov/rubocop#4513]
I find the following method to write compares with numeric values a very natural way to read/write:

```ruby
if 3 < a
if a < 5
if 3 < a && a < 5
```

The YodaCondition cop triggers on the `3 < a` syntax. The suggestion/autocorrection for the last one is `a > 3 && a < 5` which I think is an unnatural way of expressing a value between 3 and 5 (IIRC "Code Complete" mentions this as well) (using `Range#cover?` would make it a bit more readable)

So in short, I'd like a config option for the YodaCondition cop to warn me against `1 == a` and `1 != a`, but leave `<`, `<=`, `>` and `>=` alone.

--------

## RuboCop version

```
$ rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.3.3 i386-linux-gnu)
```

**Comment by bbatsov** [comment]
Sounds reasonable to me.

**Comment by smakagon** [comment]
I'll try to take a look on that. Never worked with Cop configuration before. Could be a good learning curve for me.

**Comment by Drenmi** [comment]
Grammatically it has the same problem. For example asking "is zero higher than the temperature?" instead of "is the temperature below zero?" 🙂 

I'm not opposed to the feature, though. I think it's a reasonable level of configuration. 🙂
